src/extension.ts

```diff
@@ -1353,12 +1350,7 @@ async function useContextKubernetes(explorerNode: explorer.KubernetesObject) {
 
 async function clusterInfoKubernetes(explorerNode: explorer.KubernetesObject) {
     const targetContext = explorerNode.metadata.context;
-    const shellResult = await kubectl.invokeAsync(`cluster-info`);
-    if (shellResult.code === 0) {
-        kubeChannel.showOutput(shellResult.stdout, `cluster-info for ${explorerNode.id}`);
-    } else {
-        vscode.window.showErrorMessage(`Failed to get cluster info: ${shellResult.stderr}`);
-    }
+    runKubectlCommandInTerminal(`cluster-info for ${targetContext}`, "cluster-info");
```

Comment:
This terminal name has a completely different pattern from other names.

Reply:
I prefer to let all commands to share the terminal.